app/app.jsx

```diff
@@ -2,14 +2,27 @@ import React from 'react';
 import './scss/style.scss';
 import 'normalize.css';
 
+/* import components */
+import Main from './containers/main.jsx';
+import Nav from './containers/nav.jsx';
+
 class App extends React.Component {
+
    render() {
+
+     const store = this.props.store;
```

Comment:
@emilyb7 I am quite busy for the next couple of hours, but would be happy to help later today. Feel free to push your broken code here. Maybe I can commit fixes and then leave review explaining all the points .. ?